package.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name": "ava",
 	"version": "1.0.0-rc.2",
-	"description": "Futuristic test runner 🚀",
+	"description": "Testing can be a slog. AVA's here to help you get it done.",
```

Comment:
This is almost perfect. Maybe:

> Testing can be a slog. AVA helps you get it done.

Maybe we could also add something about AVA making it more fun / faster / less painful to do tests.

Reply:
I had never heard about the word `slog` before. Is there any other more common word we could use?

> Writing tests can be boring. AVA helps you get it done.

> Testing can be a bore. AVA helps you get it done.
> Testing can be a pain. AVA helps you get it done.
> Testing can be a drag. AVA helps you get it done.

If not, `slog` is fine. Just thought I'd bring it up.

Reply:
Isn't it a bit negative? Like the first thing you want me to think about is that testing is painful? I guess what you're going for is empathizing with the user. But I think it could be done with a more positive tone.

> AVA makes testing quick and painless

Reply:
Jest used to have "painless". They're using "delightful" now. Mocha is "Simple, flexible, fun."

Perhaps I'm cynical but to me testing is something I need to get done. I may get some satisfaction out of covering difficult scenarios, but I don't need it to be fun or delightful.

Reply:
I like "can be a drag," by the way.

Reply:
How about "chore"?

> Testing can be a chore. AVA helps you get it done.

Chores aren't necessarily fun, but they're something everyone knows they have to do.